be/src/storage/delta_writer.cpp

```diff
@@ -197,7 +197,6 @@ Status DeltaWriter::_init() {
     }();
 
     // build tablet schema in request level
-    auto tablet_schema_ptr = _tablet->tablet_schema();
     RETURN_IF_ERROR(_build_current_tablet_schema(_opt.index_id, _opt.ptable_schema_param, _tablet->tablet_schema()));
```

Comment:
this might be a code bug introduced by other PR, the intention is to use tablet_schema_ptr to replace `_tablet->tablet_schema()`